packages/hmr-plugin/src/symbols.ts

```diff
@@ -37,17 +51,18 @@ export interface NgxsHmrOptions {
    * (default: 100ms)
    */
   deferTime?: number;
+
+  /**
+   * @description
+   * Add a handler which is executed when the current module code is replaced.
+   * (default: null)
+   */
+  dispose?: DisposeHook;
```

Comment:
This was a fix PR, and now it has grown into a feature PR with additional options and exported API.
Please scale this PR back to just be the fix and add these new features as a separate PR... unless these are essential for the fix?

Reply:
revert